packages/cairo.rb

```diff
@@ -7,32 +8,17 @@ class Cairo < Package
   source_url 'https://www.cairographics.org/releases/cairo-1.14.12.tar.xz'
   source_sha256 '8c90f00c500b2299c0a323dd9beead2a00353752b2092ead558139bd67f7bf16'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/cairo-1.14.12-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/cairo-1.14.12-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/cairo-1.14.12-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/cairo-1.14.12-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: 'fb449a39c94abb12d7997cb24bf57486b6e8a3baf7599484370adedffae0ecb0',
-     armv7l: 'fb449a39c94abb12d7997cb24bf57486b6e8a3baf7599484370adedffae0ecb0',
-       i686: 'bbd77b10c0d2a2a5e53046bf40659d896b84b9194d9e1065ed82f15dd584ee2a',
-     x86_64: 'ced7600e3eadce92494d7be258c9bbb4f2d4975350129470f5eff882a4a205dd',
-  })
-
   depends_on 'libpng'
   depends_on 'pixman'
   depends_on 'fontconfig' # pango requires cairo with fontconfig
+  depends_on 'gobject_introspection'
+  depends_on 'mesa'
 
   def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-xlib'
+    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} --enable-xlib --enable-xlib-xcb --enable-glesv2"
```

Comment:
Nice!  Can you please keep the separation of one configure option per line?  It just makes it easier to examine and add/remove options in the future.

Reply:
I got error if I do that. I don't know why

Reply:
You need a trailing comma on each line or you can concatenate with a trailing `\`.  Also, you need all the crew constants to be enclosed in double quotes or the variables won't expand.  Hope that helps.

Reply:
Please change to:
```
system "./configure \
  --prefix=#{CREW_PREFIX} \
  --libdir=#{CREW_LIB_PREFIX} \
  --enable-xlib \
  --enable-xlib-xcb \
  --enable-glesv2"
```